model/host/hostutil.go

```diff
@@ -1306,8 +1315,30 @@ func (h *Host) FetchProvisioningScriptUserData(settings *evergreen.Settings) (*u
 		fmt.Sprintf("--working_dir=%s", h.Distro.BootstrapSettings.JasperBinaryDir),
 	}, " ")
 
+	cmds := []string{
+		fetchClient,
+		fixClientOwner,
+		fetchScriptCmd,
+	}
+
+	if h.Distro.IsWindows() {
+		for i := range cmds {
+			cmds[i] = fmt.Sprintf("%s -l -c %s", h.Distro.ShellBinary(), util.PowerShellQuotedString(cmds[i]))
+		}
+	} else {
+		shellPrefix := []string{"set -o errexit", "set -o verbose"}
+		cmds = append(shellPrefix, cmds...)
+	}
+
+	var directive userdata.Directive
+	if h.Distro.IsWindows() {
```

Comment:
Can we combine this with the if/else above?

Reply:
Done.